modules/loggers/mod_log_config.c

```diff
@@ -961,7 +968,7 @@ static char *parse_log_misc_string(apr_pool_t *p, log_format_item *it,
      */
     it->arg = apr_palloc(p, s - *sa + 1);
 
-    d = it->arg;
+    d = (char *)it->arg;
```

Comment:
`d = apr_palloc(...); it->arg = d;` or `it->arg = d = apr_palloc(...);` would avoid the const away cast.

Reply:
fixed